scripts/download_modulemd.sh

```
@@ -0,0 +1,166 @@
+            minimal_docs.append(doc)
+
+    with open('${temp_file}', 'w') as f:
+        yaml.dump_all(minimal_docs, f, default_flow_style=False, sort_keys=False)
+
+    sys.exit(0)
+except Exception as e:
+    print(f"Error: {e}", file=sys.stderr)
+    sys.exit(1)
+PYTHON_EOF
+
+    if [ $? -eq 0 ]; then
```

Shellcheck (0.11.0)

[style] Check exit code directly with e.g. 'if mycmd;', not indirectly with $?.

(SC2181)